### ISSUE TITLE
ValueError: math domain error

### ISSUE THREAD

**Issue by DanyaSWorlD** [DogsTailFarmer/martin-binance#27]
```[INFO][ProcID 6]! debug output:
[INFO][ProcID 6]! sum_amount_first: 0, sum_amount_second: 0
[INFO][ProcID 6]! part_amount_first: 0, part_amount_second: 0
[INFO][ProcID 6]! initial_first: 26, initial_second: 0.00004
[INFO][ProcID 6]! initial_reverse_first: 0, initial_reverse_second: 0
[INFO][ProcID 6]! reverse_init_amount: 0
[INFO][ProcID 6]! reverse_target_amount: 0
[INFO][ProcID 6]! correction_amount_first: 0, correction_amount_second: 0
[INFO][ProcID 6]! tp_order: ()
[INFO][ProcID 6]Traceback (most recent call last):
[INFO][ProcID 6]! tp_part_amount_first: 0, tp_part_amount_second: 0
[INFO][ProcID 6]  File "C:\Users\admin\AppData\Local\Packages\BED1B04E-C73B-45D2-BB43-2E8EB15322B6_j4y6f2x461yhe\LocalState\WinPython\python-3.8.5.amd64\Lib\site-packages\martin_binance\executor.py", line 1375, in place_grid
[INFO][ProcID 6]! profit_first: 0, profit_second: 0
[INFO][ProcID 6]    amount_first_grid = self.set_trade_conditions(buy_side,
[INFO][ProcID 6]! part_profit_first: 0, part_profit_second: 0
[INFO][ProcID 6]  File "C:\Users\admin\AppData\Local\Packages\BED1B04E-C73B-45D2-BB43-2E8EB15322B6_j4y6f2x461yhe\LocalState\WinPython\python-3.8.5.amd64\Lib\site-packages\martin_binance\executor.py", line 1782, in set_trade_conditions
[INFO][ProcID 6]! deposit_first: 26.28, deposit_second: 0
[INFO][ProcID 6]    q_max = int(math.log(1 + (depo_c - amount_first_grid) * self.martin * (self.martin - 1) / amnt_2, self.martin))
[INFO][ProcID 6]! command: None
[INFO][ProcID 6]ValueError: math domain error
[INFO][ProcID 6]! reverse: False```

**Comment by DanyaSWorlD** [comment]
1.2.2, on v 1.2.6 it's error under issue #28

**Comment by DogsTailFarmer** [comment]
Fixed on 1.2.6
